### PR TITLE
LambdaTest Quickstart

### PR DESCRIPTION
**Pull request by prateeksainilambdatest**
# Summary

The integration of LambdaTest with New Relic offers a seamless experience for users aiming to optimize and monitor their stats during automated test execution. By bridging the gap between testing and performance monitoring, this integration brings forward an out-of-the-box dashboard that enables users to visualize and understand their testing metrics in real-time, set against the backdrop of broader application performance insights offered by New Relic.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/lambdatest/config.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+id: f5d5d638-24ee-4a1c-859f-9cbd4d8cabbc
```
Comment (aswanson-nr) [inline quickstarts/lambdatest/config.yml:1]: Please remove this line, new quickstarts will get an ID when they are merged into the `main` branch for the first time.
```suggestion
```

File: quickstarts/lambdatest/config.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  - cross-browser
+  - featured
+​
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+​
+installPlans:
+  - third-party-lambdatest
+dataSourceIds:
+  - lambdatest
```
Comment (aswanson-nr) [inline quickstarts/lambdatest/config.yml:56]: This data source does not appear to exist, could you add it to this pull request?

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @prateeksainilambdatest, this looks like it's coming together! I just found a few spots that need to be cleaned up. 

We will need to adjust these screenshots for the dashboard as well, they need to be in a 16:9 aspect ratio to properly fit on the screen.

File: quickstarts/lambdatest/config.yml (hunk 1 above)
Comment (prateeksainilambdatest) [reply quickstarts/lambdatest/config.yml]: Removed

File: quickstarts/lambdatest/config.yml (hunk 2 above)
Comment (prateeksainilambdatest) [reply quickstarts/lambdatest/config.yml:54]: data source added

File: quickstarts/lambdatest/config.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+keywords:
+  - lambdatest
+  - logs
+  - newrelic partner
+  - devops
+  - cicd
+  - testing
+  - cross-browser
+  - featured
+
```
Comment (jcountsNR) [inline quickstarts/lambdatest/config.yml:47]:
```suggestion
  - NR1_addData
```

File: dashboards/lambdatest/lambdatest-logs.json [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "variables": []
+}
```
Comment (jcountsNR) [inline dashboards/lambdatest/lambdatest-logs.json:543]: The dashboard sanitization script needs to be ran for this. You can follow the steps in the [contributing guide](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) to run it. Other than that it looks great!

**Review by jcountsNR — CHANGES_REQUESTED** [review]

File: quickstarts/lambdatest/config.yml (hunk 3 above)
Comment (prateeksainilambdatest) [reply quickstarts/lambdatest/config.yml:48]: Added

File: dashboards/lambdatest/lambdatest-logs.json (hunk 4 above)
Comment (prateeksainilambdatest) [reply dashboards/lambdatest/lambdatest-logs.json:508]: Updated dashboard.

**Review by jcountsNR — APPROVED** [review]

File: quickstarts/lambdatest/config.yml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  - cicd
+  - testing
+  - cross-browser
+  - featured
+  - NR1_addData
+  
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+
+installPlans:
```
Comment (jcountsNR) [inline quickstarts/lambdatest/config.yml:52]:
```suggestion

```

File: quickstarts/lambdatest/config.yml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  - third-party-lambdatest
```
Comment (jcountsNR) [inline quickstarts/lambdatest/config.yml:53]:
```suggestion

```

**Review by jcountsNR — CHANGES_REQUESTED** [review]
The install plan needs to be removed

File: data-sources/lambdatest/config.yml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+id: lambdatest
+displayName: LambdaTest
+description: |
+  LambdaTest is an AI-powered test orchestration and execution platform to run manual and automated tests at scale. With LambdaTest, developers and testers can test their websites and mobile applications across 3000+ real browsers, devices, and operating systems.
```
Comment (jcountsNR) [inline data-sources/lambdatest/config.yml:4]: @prateeksainilambdatest 

This has to be under 240 characters, it is currently 263. Can you condense this description a bit so it fits correctly in the tile?

**Review by jcountsNR — CHANGES_REQUESTED** [review]

File: quickstarts/lambdatest/config.yml (hunk 5 above)
Comment (prateeksainilambdatest) [reply quickstarts/lambdatest/config.yml]: installPlans removed

File: data-sources/lambdatest/config.yml (hunk 7 above)
Comment (prateeksainilambdatest) [reply data-sources/lambdatest/config.yml]: Updated

**Review by jcountsNR — APPROVED** [review]
I'll approve from my end. @prateeksainilambdatest , can you make sure the CLA is signed? It looks like that is the last check required.

**Comment by jcountsNR** [comment]
Hi @aswanson-nr , this one looks good to go to me, I approved it on my end.

**Review by aswanson-nr — APPROVED** [review]
Looks good from my end! We'll get this merged early next week.

File: data-sources/lambdatest/config.yml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+id: lambdatest
+displayName: LambdaTest
+description: |
+  LambdaTest is an AI-powered test orchestration and execution platform to run manual and automated tests at scale.
+install:
+  primary:
+    link:
+      url: https://www.lambdatest.com/support/docs/new-relic-integration/
```
Comment (pkosanam) [inline data-sources/lambdatest/config.yml:8]: Hi @rahulbasu-nr ,
can users be redirected outside of New Relic ?

**Review by pkosanam — COMMENTED** [review]
Kindly check the datasource URL.

**Comment by aswanson-nr** [comment]
> Looks good from my end! We'll get this merged early next week.

I was just talking about the next time we would run through and merge PRs! Feel free to get this merged when ready. :)

**Pull request merged**